app/serializers/activitypub/actor_serializer.rb

```diff
@@ -173,12 +164,30 @@ def name
       "##{object.name}"
     end
 
-    def statuses_count
-      object.statuses_count.to_s
+    def featured
+      true
+    end
+  end
+
+  class AccountSerializer < ActivityPub::Serializer
+    include RoutingHelper
+
+    attributes :type, :id, :name, :featured
+
+    def type
+      'Accoount'
```

Comment:
@nightpool Your suggestion is actually a bit problematic because `Account` is not an ActivityPub type. There is `Person`, `Service`, etc... `Mention` is what comes up in `tag` usually. Perhaps this is where `Endorsement` would appear as a type.

Reply:
It has been made to reflect the `actor_type` of the account.